langgraph_bot/main.py

```
@@ -22,7 +22,21 @@ def run_entire_flow():
             # "name":post['source_name'],     #   yet to be discussed
         }
 
-        all_posts.append(g.invoke(initial_state))
+        try:
+            result = g.invoke(initial_state)
+        except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)